icons/amazonsimpleemailservice.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Amazon Simple Email Service</title><path d="M20.1429 21c0 1.1571-.9858 2.1429-2.1429 2.1429S15.8571 22.157 15.8571 21c0-1.0286.7715-1.8857 1.7143-2.1v-1.3286h-5.1428v2.1857c.9857.2143 1.7143 1.0715 1.7143 2.1C14.1429 23.0143 13.157 24 12 24s-2.1429-.9857-2.1429-2.1429c0-1.0285.7715-1.8857 1.7143-2.1v-2.1857H6.4286V18.9c.9857.2143 1.7143 1.0714 1.7143 2.1 0 1.1571-.9858 2.1429-2.1429 2.1429S3.8571 22.157 3.8571 21c0-1.0286.7715-1.8857 1.7143-2.1v-1.7571c0-.2572.1715-.4286.4286-.4286h5.5714v-2.5714H6c-.2571 0-.4286-.1715-.4286-.4286V6c0-.2571.1715-.4286.4286-.4286h12c.2571 0 .4286.1715.4286.4286v7.7143c0 .2571-.1715.4286-.4286.4286h-5.5714v2.5714H18c.2571 0 .4286.1714.4286.4286V18.9c.9857.2143 1.7143 1.0714 1.7143 2.1zM12 20.5714c-.6857 0-1.2857.6-1.2857 1.2857 0 .6858.6 1.2858 1.2857 1.2858.6857 0 1.2857-.6 1.2857-1.2858 0-.6857-.6-1.2857-1.2857-1.2857zm-6-.8571c-.6857 0-1.2857.6-1.2857 1.2857 0 .6857.6 1.2857 1.2857 1.2857.6857 0 1.2857-.6 1.2857-1.2857 0-.6857-.6-1.2857-1.2857-1.2857zM24 12c0 2.5286-.7714 4.9714-2.2714 6.9857l-.6857-.4714c1.3714-1.9286 2.1-4.1572 2.1-6.5143C23.1429 5.8714 18.1286.8571 12 .8571 5.8714.8571.8571 5.8714.8571 12c0 2.3571.7286 4.5857 2.1 6.5143l-.6857.4714C.7714 16.9714 0 14.5286 0 12 0 5.4 5.4 0 12 0s12 5.4 12 12ZM6.4286 6.9429v5.8285l3.3-3zm10.4143-.5143H7.157L12 10.5857Zm.7285 6.3428V6.943l-3.3 2.8285zm1.7143 8.2715c0-.7286-.6-1.2858-1.2857-1.2858-.6857 0-1.2857.5572-1.2857 1.2858 0 .6857.6 1.2857 1.2857 1.2857.6857 0 1.2857-.6 1.2857-1.2857zM7.1143 13.3286h9.7714l-3.3-2.9572L12.3 11.4857q-.1286.1286-.3.1286t-.2571-.1286l-1.3286-1.1143z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Amazon Simple Email Service</title><path d="M11.9996 0C5.3833 0 0 5.3834 0 11.9998c0 2.5316.7813 4.9544 2.2599 7.0051l.6955-.5014C1.5827 16.5993.8571 14.3505.8571 11.9998.8571 5.856 5.856.8572 12.0004.8572c6.144 0 11.1425 4.999 11.1425 11.1426 0 2.3508-.7256 4.5995-2.0983 6.5037l.6955.5014C23.2187 16.9542 24 14.5314 24 11.9998 24 5.3834 18.6163 0 11.9996 0z"/><path d="M6 16.7142a.4285.4285 0 0 0-.4286.4285v1.7598c-.9643.2048-1.7143 1.0822-1.7143 2.0974 0 1.1615.9815 2.143 2.1429 2.143s2.1429-.9815 2.1429-2.143c0-1.0152-.75-1.8926-1.7143-2.0974v-1.3312h5.1428v2.1883c-.9643.2049-1.7143 1.0822-1.7143 2.0975C9.8571 23.0186 10.8386 24 12 24s2.1429-.9814 2.1429-2.1429c0-1.0153-.75-1.8926-1.7143-2.0975v-2.1883h5.1428v1.3312c-.9643.2048-1.7143 1.0822-1.7143 2.0974 0 1.1615.9815 2.143 2.1429 2.143s2.1429-.9815 2.1429-2.143c0-1.0152-.75-1.8926-1.7143-2.0974v-1.7598A.4285.4285 0 0 0 18 16.7142h-5.5714v-2.5715H18c.237 0 .4286-.192.4286-.4286V5.9997A.4285.4285 0 0 0 18 5.571H6a.4285.4285 0 0 0-.4286.4286v7.7144c0 .2366.1916.4286.4286.4286h5.5714v2.5715H6zm1.2857 4.2857c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857S7.2857 20.3031 7.2857 21zm12 0c0 .697-.5889 1.2858-1.2857 1.2858s-1.2857-.5889-1.2857-1.2858c0-.6968.5889-1.2857 1.2857-1.2857s1.2857.5889 1.2857 1.2857zm-1.7143-8.248L14.259 9.7703l3.3124-2.8389v5.8205zm-.7298-6.3236-4.842 4.1499-4.8412-4.15h9.6832zm-10.413.5031L9.741 9.7707 6.4286 12.752V6.9314zm.6878 6.3541 3.2807-2.9525 1.3239 1.135a.4253.4253 0 0 0 .2786.1032.4253.4253 0 0 0 .2785-.1033l1.3243-1.1349 3.2812 2.9525H7.1164zM12 20.5714c.6968 0 1.2857.5888 1.2857 1.2857 0 .6969-.5889 1.2857-1.2857 1.2857s-1.2857-.5888-1.2857-1.2857c0-.6969.5889-1.2857 1.2857-1.2857z"/></svg>
```
I am also seeing [differences in image](https://github.com/simple-icons/simple-icons/assets/35000807/8a8a3f7e-6d96-47c0-a372-819ea5eb34f2)


This should fix it.